### PR TITLE
Overlay in schedule settings

### LINKED ISSUE #166: Settings page show selected color overlay over the title
**Describe the bug**
When accessing the schedule settings page, the color of the selected month go over the title (as seen on the video below).

**To Reproduce**
1. Go to schedule.
2. Click on settings.
3. See that the selected color is above the title.

**Expected behavior**
The selected color should disappear at the same time that the text disappears from the rollup.

**Screenshots**
https://user-images.githubusercontent.com/25663435/126724534-e903d230-7538-41e7-8910-e8e1000a93b4.mp4

**Smartphone (please complete the following information):**
 - Device: samsung S10e
 - OS: Android
 - Version: 11

### PR DESCRIPTION
**Pull request by camillebrulotte**
https://user-images.githubusercontent.com/37625944/126870482-6370029d-00d4-449f-91b5-4bce905f512d.mp4

https://user-images.githubusercontent.com/37625944/126870510-d87b2c66-d90c-4ada-be4e-6530abbd0535.mp4

Closes #166

### CODE REVIEW

File: lib/core/utils/utils.dart [hunk 1]
```diff
@@ -31,4 +32,11 @@ class Utils {
       Fluttertoast.showToast(msg: intl.no_connectivity);
     }
   }
+
+  static Color getColorByBrightness(
```
Comment (AlexandreMarkus) [inline lib/core/utils/utils.dart:36]: Excellent 👌

**Review by AlexandreMarkus — APPROVED** [review]

**Comment by MysticFragilist** [comment]
@apomalyn fyi, this is what the [CI_UPDATE_GOLDENS] bot commit everytime we launch it:
<details>
  <summary>Git diff of commit 756c313</summary>

</details>

For now it's not really critical, if you want to concentrate on the deployment, I don't have a problem, I'll make sure to update goldens in future PR! 😄

**Review by MysticFragilist — APPROVED** [review]
LGTM 😄

**Review by JonathanDuvalV — APPROVED** [review]
LGTM!

**Pull request merged**